onnxruntime/python/tools/transformers/models/llama/requirements.txt

```diff
@@ -1,7 +1,9 @@
 optimum>=1.14.1
 transformers>=4.33.2,<= 4.37.2
 torch>=2.2.0
-onnx==1.16.1
+# git+http://github.com/onnx/onnx.git@84051888d0943883a0edbf683f68c05ca3b28c40#egg=onnx
+--extra-index-url https://pypi.org/simple
```

Comment:
We are not allowed to use --extra-index-url

Reply:
this is just to validate onnx 1.16.2rc which can be located at the link. onnx 1.16.2rc will be published once this test passes. I will replease the related changes to onnx==1.16.2 before merge this PR.

Reply:
> Why cannot we build it from source?

The main propose is to test out the package so I commented out the line for building from source